upgrades/cdk-erigon_fep-to-pp.md

```diff
@@ -95,18 +78,19 @@ This process may take a couple hours to complete, but downtime from the point of
       zkevm.mock-witness-generation: true
       zkevm.disable-virtual-counters: true
       ```
-   4. Update aggkit config with:
-   6. Stop the following components:
+   4. Stop the following components:
       1. dac
       2. sequence-sender
       3. aggregator
       4. executors
       5. provers
       6. pool-manager
 4. **Migrate to PP**:
-   1. Request Polygon (as the RollupManager Admin) to send the transaction to perform the migration: `cast send --private-key ${ADMIN_PKEY} $ROLLUP_MANAGER "initMigration(uint32,uint32, bytes)" ${ROLLUPID} ${ROLLUPTYPEID} 0x`
-   2. Wait until the transaction is finalized.
-5. **Start aggsender**:
+   1. Request Polygon (as the RollupManager Admin) to send the transaction to perform the migration: `cast send --private-key ${ADMIN_PKEY} $ROLLUP_MANAGER "initMigration(uint32,uint32,bytes)" ${ROLLUPID} ${ROLLUPTYPEID} 0x06e76665`
```

Comment:
```suggestion
   1. Request Polygon (as the AgglayerManager Admin) to send the transaction to perform the migration: `cast send --private-key ${ADMIN_PKEY} $AGGLAYER_MANAGER "initMigration(uint32,uint32,bytes)" ${ROLLUPID} ${ROLLUPTYPEID} 0x06e76665`
```

Reply:
Additionally, we would need to update all occurrences of RollupManager to AgglayerManager